### DISCUSSION TITLE
早安

### DISCUSSION THREAD

**Discussion by shiliuyexiafeng** [share121/inter-knot#3012] [其他]
![IMG_20240819_115514](https://github.com/user-attachments/assets/b4ab1da0-a471-488e-a862-3a6a34412cbb)

**Comment by lllliufeng** [comment]
午安
还有图片**没传完**吧

**Comment by shiliuyexiafeng** [comment]
没上传成功吗<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_4941628593528507734.png" alt="阿姬倒地了03" width="75" >

**Reply by lllliufeng** [reply]
要等\![Uploading XXX…](\)变成\![XXX](https://github.com/user-attachments/assets/XXX\)哦~

**Comment by shiliuyexiafeng** [comment]
好的，下次会注意<img src="https://upload-bbs.mihoyo.com/upload/2022/09/20/80a7ef5a067a1acd495fcc09140b069c_5444302550606416083.gif" alt="爱莉希雅-比心" width="75" >
